org.mavlink.QGroundControl.yml

```diff
@@ -0,0 +1,45 @@
+app-id: org.mavlink.QGroundControl
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: QGroundControl
+rename-desktop-file: org.mavlink.qgroundcontrol.desktop
```

Comment:
why not using `org.mavlink.qgroundcontrol` as the app-id?

Reply:
I thought about it but I think that `QGroundControl` is easier readable.

Would you strongly recommend keeping the original lower-case AppID?

Reply:
Use upstream app-id.